src/egon/data/datasets/DSM_cts_ind.py

```diff
@@ -460,16 +463,24 @@ def ind_sites_data_import():
     Import industry sites data necessary to identify DSM-potential.
     """
     # calculate timeseries per site
+    scenarios = config.settings()["egon-data"]["--scenarios"]
 
     # scenario eGon2035
-    dsm_2035 = calc_ind_site_timeseries("eGon2035")
-    dsm_2035.reset_index(inplace=True)
+    if "eGon2035" in scenarios:
+        dsm_2035 = calc_ind_site_timeseries("eGon2035")
+        dsm_2035.reset_index(inplace=True)
     # scenario eGon100RE
-    dsm_100 = calc_ind_site_timeseries("eGon100RE")
-    dsm_100.reset_index(inplace=True)
-    # bring df for both scenarios together
-    dsm_100.index = range(len(dsm_2035), (len(dsm_2035) + len((dsm_100))))
-    dsm = dsm_2035.append(dsm_100)
+    if "eGon100RE" in scenarios:
```

Comment:
These if and elif statements are not that easy to understand. It would be helpful if it would work with fewer options. If that is not possible, some short comments why it is done could be helpful.

Reply:
The if statements are now simplified in e212ef0a986e6dba20d8b2745b67d213f171f467